homeassistant/components/ozw/climate.py

```diff
@@ -199,16 +200,18 @@ def fan_modes(self):
     @property
     def temperature_unit(self):
         """Return the unit of measurement."""
-        if self.values.temperature is not None and self.values.temperature.units == "F":
-            return TEMP_FAHRENHEIT
-        return TEMP_CELSIUS
+        return self._convert_units(self.values.temperature.units)
```

Comment:
https://github.com/home-assistant/core/pull/45330#discussion_r560278575

Reply:
@MartinHjelmare  - I need a bit of guidance for your comment on the other PR.  Is there more to it then just using ```self._current_mode_setpoint_values```?

Reply:
Probably use the `units` attribute of the setpoint value in `self._current_mode_setpoint_values`.

Reply:
I wonder if we should also use this for https://github.com/home-assistant/core/pull/45369/files#diff-c4fa29871316d2613f37a19a45137f18faab85ee6a3b2ea31bffe2410775625aR212?